docs/transport/s3.md

```diff
@@ -0,0 +1,28 @@
+# transport: s3
+
+The S3 transporter puts logical WAL messages as *gzipped* objects in an S3 bucket partitioned by year, month and day in increasing order of time.
+
+## Key Construction
+
+The basic construction of an S3 key for this transporter is defined as:
+```
+<bucket name>/<key space>/<year as YYYY>/<month as MM>/<day as DD>/<datetime>_<first wal record id>.gz
+```
+
+### Example
+
+For a given configuration:
```

Comment:
Can you also make a table of CLI args like in the main readme. I think it may be helpful.

Reply:
Done.

Reply:
I think it's redundant and repeats "code". There's a help function for this. I don't think any particular options needs elaboration beyond what's already there.